core/src/main/java/io/questdb/griffin/SqlOptimiser.java

```diff
@@ -3097,6 +3097,8 @@ private void optimiseOrderBy(QueryModel model, int topLevelOrderByMnemonic) {
                 // Set artificial limit to trigger LimitRCF use, so that parent models don't use the followedOrderByAdvice flag and skip necessary sort
                 // Currently the only way to delineate order by advice is through use of factory that returns false for followedOrderByAdvice().
                 // TODO: factories should provide order metadata to enable better sort-skipping
+                // TODO: action what the hell is that concat?!
```

Comment:
?

Reply:
Oh, I see `"" + Long.MAX_VALUE` should be replaced with a constant with the string `"9223372036854775807"`.

Reply:
Replaced with constant.